README.md

```diff
@@ -142,15 +144,16 @@ In order to update all the vendored libraries in the `/vendor` directory.
 
 Once you've made the update to the codebase that you'd like to submit, you may [create a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) to the opentelemetry-php project.
```

Comment:
Could you please also reduce the visual noise of this? eg.:
"If you want to submit changes to the codebase, please create a pull request"